components/authentication/org.wso2.carbon.analytics.auth.rest.api/src/gen/java/org/wso2/carbon/analytics/auth/rest/api/dto/UserDTO.java

```diff
@@ -35,6 +35,9 @@
     @JsonProperty("partialRefreshToken")
     private String partialRefreshToken = null;
 
+  @JsonProperty("validityPeriod")
```

Comment:
Seems like this is not properly indented. Please make sure that all changes are properly formatted.